modules/test/playwright/tests/portal-search-web/searchFacets.spec.ts

```diff
@@ -0,0 +1,209 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {searchPageTest} from '../../fixtures/searchPageTest';
+
+export const test = mergeTests(loginTest(), searchPageTest);
+
+test.describe('LPD-19994 Clear and retain facet selections', () => {
```

Comment:
For including ticket numbers, we can use tags. These should also be placed on the individual tests themselves instead of the entire test `describe` group in case more tests are added for other tickets.

For example: 
```
test('clears facet terms after new keyword search @LPD-19994', async ({
```

Reply:
Got it! Moving the tag to the individual tests 👌